### ISSUE TITLE
Always receive alerts of health monitor job failures

### ISSUE THREAD

**Issue by mogul** [18F/cg-product#17]
In order to ensure we're aware and able to respond to problems, we want alerts from the BOSH health monitor to be routed through Riemann.
## Acceptance criteria:
- [ ] When a bosh job fails, alerts propagate from health monitor to riemann to pagerduty
- [ ] System metrics from bosh heartbeat events propagate to riemann, optionally trigger alerts when bad

Pull request: https://github.com/cloudfoundry/bosh/pull/1337

**Comment by rogeruiz** [comment]
Health monitor knows a service has failed, but with logging from bosh monit we'd have insight into these failures.

**Comment by jmcarp** [comment]
Is there more to this than sending health monitor alerts to pagerduty / cloudwatch?

https://bosh.io/docs/hm-config.html

**Comment by rogeruiz** [comment]
@jmcarp Please link stuff on here.

**Comment by jmcarp** [comment]
The latest work is at https://github.com/sharms/bosh--we'll send a PR upstream once we confirm that everything's working.

**Comment by mogul** [comment]
Just waiting on the upstream release now before we can push this forward again.

**Comment by jmcarp** [comment]
Deferring until our riemann plugin for health monitor is released: see https://github.com/cloudfoundry/bosh/pull/1351#issuecomment-249616312.

**Comment by jmcarp** [comment]
Now that bosh has released our changes, this Just Works. The alerts might be too chatty, but I think we can deal with that in a separate task.

**Comment by rogeruiz** [comment]
This works. and is very chatty. We should file a separate story to make the pages less annoying.
